compiler+runtime/include/cpp/jank/runtime/module/loader.hpp

```diff
@@ -68,6 +76,24 @@ namespace jank::runtime::module
       option<file_entry> cljc;
     };
 
+    struct find_result
+    {
+      /*
+       * All the sources for a module
+       */
+      entry sources;
+      /*
+       * On the basis of origin, source that should be loaded.
+       */
+      option<module_type> to_load;
+
+      find_result(entry const &sources, module_type const to_load)
+        : sources{ sources }
+        , to_load{ to_load }
+      {
+      }
```

Comment:
No need for this core. Aggregate initialization will work the same without it. By having it, though, you prevent aggregate initialization.